CHANGES.rst

```diff
@@ -27,7 +27,7 @@ Enhancements
   as a Python function.
 * ``Equal[]`` now compares complex against other numbers properly.
 * Improvements in handling products with infinite factors: ``0 Infinity``-> ``Indeterminate``, and ``expr Infinity``-> ``DirectedInfinite[expr]``
-* `StringSplit` not support lists as a first argument.
+* `StringSplit` now supports lists as a first argument.
 * ``SetDelayed`` now accept several conditions impossed both at LHS as well as RHS.
```

Comment:
Sigh - just noticed that this should be two backquotes not one. (This kind of stuff I'd probably catch in reviewing release notes though).